start.py

```diff
@@ -21,6 +22,12 @@
 
 import xml.etree.ElementTree
 
+release = lsb_release.get_lsb_information()
```

Comment:
in 20.04, this function doesn't exists. I have a `get_distro_information` that seems to return the same type of information.